etc/env/example.env

```diff
@@ -113,22 +113,22 @@ TRAEFIK_SOCKET=unix:///var/run/docker.sock
 MINIO_VERSION=latest
 MINIO_UI_PORT=9090
 MINIO_PORT=9000
-MINIO_PUBLIC_URL=https://${CANDIG_DOMAIN}:${MINIO_PORT}
+MINIO_PUBLIC_URL=http://${CANDIG_DOMAIN}:${MINIO_PORT}
 MINIO_PRIVATE_URL=http://minio:9000
 MINIO_BUCKET=samples
 MINIO_REGION=us-east-1
 MINIO_DATA_DIR=/data
 
 # set to 1 if using SSL via self-signed certificate
-MINIO_SELF_CERT=1
+MINIO_SELF_CERT=0
```

Comment:
is this the best setting for standard deployments, or just for a particular deployment?